CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### Added
 
+* SSO integration using the nextjs-auth0 package
+* Middleware that conditionally loads the Auth0 wrapper based on an
```

Comment:
I would probably put this second point under an `### Internal` heading, since it's not something users or managers are likely to care much about.

Reply:
done!